src/commands/protocols/publish/receiver/v1.0.0/v1-0-0-handle-store-request-command.js

```diff
@@ -31,7 +45,7 @@ class HandleStoreRequestCommand extends HandleProtocolMessageCommand {
         await this.operationIdService.updateOperationIdStatus(
             operationId,
             blockchain,
-            OPERATION_ID_STATUS.VALIDATE_ASSET_REMOTE_START,
+            OPERATION_ID_STATUS.PUBLISH.PUBLISH_VALIDATE_ASSET_REMOTE_START,
```

Comment:
Move this to line 52, to so it measure validation and add another event for reading from catch